File: TodoDataAPI/Models/Todo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace TodoDataAPI.Models
{
    public class Todo
    {
        [Key]
        public int TodoID { get; set; }

        [Required]
        
        public string Title { get; set; }

        
        public string Description { get; set; }

        public DateTime? CreatedAt { get; set; }


        public bool Done { get; set; } = false;



    }
}

File: todo-front/src/app/models/todo.ts
export class Todo {
  private todoID: string;
  private title: string;
  private description: string;
  private createdAt: string;
  private done: boolean;

  constructor(todoID: string, title: string, description: string, createdAt: string, done: boolean) {
    this.todoID = todoID;
    this.title = title;
    this.description = description;
    this.createdAt = createdAt;
    this.done = done;
  }
}
